dml_events.go

```diff
@@ -253,84 +250,77 @@ func NewBinlogDMLEvents(table *schema.Table, ev *replication.BinlogEvent, pos my
 	}
 }
 
-func loadColumnsForTable(table *schema.Table, valuesToVerify ...RowData) ([]string, error) {
```

Comment:
Since I'm passing down `[]TableColumn` anyway, I might as well just do the column name quoting in the (`build*` methods that actually builds the query).

This method is a bit vague on what it does anyway, so I removed it.